pre_submit.csh

```diff
@@ -217,7 +217,7 @@ cd -
 
 echo "If this is the start of a new month, issue a pull request to DART_CASES:"
 echo '   % git status -uno | sed -e "s#modified:#git add#" >! push_prep.csh'
-echo '   Edit push_prep.csh to make it `git add` all of the modified files'
+echo '   Edit push_prep.csh to make it a csh script '
```

Comment:
You actually don't need to edit it to make it a csh script. Because you give the filename as an argument to csh, it automatically gets interpreted with csh syntax. Now - I'd still prefer you edit the file to examine the contents ...